Code/Framework/AzToolsFramework/Tests/FocusMode/EditorFocusModeTests.cpp

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzTest/AzTest.h>
+#include <AzCore/UserSettings/UserSettingsComponent.h>
+#include <AzCore/Component/TransformBus.h>
+#include <AzToolsFramework/FocusMode/FocusModeInterface.h>
+#include <AzToolsFramework/UnitTest/AzToolsFrameworkTestHelpers.h>
+
+namespace AzToolsFramework
+{
+    class EditorFocusModeTests
+        : public ::testing::Test
```

Comment:
suggestion: This is likely handled by the environment but it might be worth inheriting from `AZ::AllocatorsTestFixture` and calling the base Setup/Teardown to ensure the allocators and environment are created (or just add a comment to say these are provided by the environment so it's clear)